Disaster Recovery/SafeHaven 4/Create-a-SafeHaven-Cluster.md

```diff
@@ -0,0 +1,74 @@
+{{{
+  "title": "Create a SafeHaven Cluster",
+  "date": "05-08-2017",
+  "author": "Sharon Wang",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Article Overview
+
+This article explains how to create a SafeHaven Cluster.
+
+### Requirements
+
+1) Pre-deployed CMS server.

```

Comment:
Change numbering from `1)` to `1.`
This will make this KB consistent with the rest of the KBs.